db/db.go

```diff
@@ -90,6 +91,17 @@ func (Record) TableName() string {
 	return "events"
 }
 
+type BlackDevice struct {
```

Comment:
Can we name this something else? I've never heard of items on a blacklist being called black items....my suggestion would be `BlacklistedDevice`